wagtailmedia/locale/de/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,380 @@
+# Translations for wagtailmedia
+# Copyright (C) 2014 Torchbox Ltd and individual contributors.
+# This file is distributed under the same license as the wagtailmedia package.
+# Translators:
+# Henrik Kröger <hedwig@riseup.net>, 2020.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: wagtailmedia\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2020-12-17 17:07+0100\n"
+"PO-Revision-Date: 2020-12-17 17:33+0100\n"
+"Last-Translator: Henrik Kröger <hedwig@riseup.net>\n"
+"Language-Team: \n"
+"Language: de\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Poedit 2.4.2\n"
+
+#: forms.py:79
+msgid "Add"
+msgstr "Hinzufügen"
+
+#: forms.py:79
+msgid "Add/edit media you own"
+msgstr "Hinzufügen/Editieren Ihrer Medien"
+
+#: forms.py:80 views/media.py:139 views/media.py:194
+msgid "Edit"
+msgstr "Bearbeiten"
+
+#: forms.py:80
+msgid "Edit any media"
+msgstr "Alle Medien bearbeiten"
```

Comment:
Without having a closer look at the context, wouldn't it be better translated with:
"Beliebige Medien bearbeiten"